src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/FlowControllerV2.java

```diff
@@ -274,4 +276,13 @@ private void verifyRequest(FlowRequestV2 request) {
         }
         return Optional.empty();
     }
+
+    @ApiOperation(value = "Creates a new flow mirror point", response = FlowMirrorPointResponseV2.class)
+    @PostMapping(path = "/{flow_id}/mirror")
+    @ResponseStatus(HttpStatus.OK)
+    public CompletableFuture<FlowMirrorPointResponseV2> createFlowMirrorPoint(
```

Comment:
We don't have a "read" method from mirror points. Without it there is no to way to make any tests for them.